### PR TITLE
MIR-857-Further_information_for_author_search

### PR DESCRIPTION
**Pull request by rochpaul**
Use lobid API for DifferentiatedPerson or CorporateBody (default author
search)

[Link to jira](https://mycore.atlassian.net/browse/MIR-857).

### CODE REVIEW

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640&bid=20217210)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Clones removed
==============
+ mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640&bid=20217210)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 1]
```diff
@@ -421,9 +573,9 @@
       $typeMenu.attr("role", "menu");
 
       for ( var type in SearchEntity.TYPES) {
-        if (SearchEntity.TYPES[type][options.searchEntityType].enabled == false)
-          continue;
-
+      	if (SearchEntity.TYPES[type][options.searchEntityType].enabled == false || type.includes("_FALLBACK"))
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:576]: Codacy found an issue: [Avoid using if statements without curly braces](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
- Added 2
           

Clones added
============
- mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js  16
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640&bid=20217210)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 2]
```diff
@@ -819,7 +989,7 @@
 
   function getTypeFromURL(url) {
     for ( var type in SearchEntity.TYPES) {
-      if (url.indexOf(SearchEntity.TYPES[type].baseURI) != -1)
+      if (url.indexOf(SearchEntity.TYPES[type].baseURI) != -1 && !type.includes("_FALLBACK"))
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:992]: Codacy found an issue: [Avoid using if statements without curly braces](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640)

**Comment by kkrebs** [comment]
Die angezeigten Informationen zum Autor dürfen nur eine Erleichterung bei der Auswahl sein. Diese dürfen auf keinen Fall in das Textfeld übernommen werden. Bzw. wenn sie übernommen werden, müsste der PostProcess diese wieder entfernen. Momentan sieht das MODS nach Übernahme wie folgt aus:
```
        <mods:name type="personal">
            <mods:displayForm>Stephan, Robert | | Informatiker</mods:displayForm>
            <mods:role>
              <mods:roleTerm authority="marcrelator" type="code">aut</mods:roleTerm>
            </mods:role>
            <mods:nameIdentifier type="gnd">1027147291</mods:nameIdentifier>
            <mods:namePart type="family">Stephan</mods:namePart>
            <mods:namePart type="given">Robert</mods:namePart>
            <mods:namePart type="given">|</mods:namePart>
            <mods:namePart type="given">|</mods:namePart>
            <mods:namePart type="given">Informatiker</mods:namePart>
        </mods:name>
```
Unschön ist auch der doppelte Trenner ( | | ) falls wie in dem Fall kein Jahr vorhanden ist.

**Comment by kkrebs** [comment]
branch also need an rebase, we like to merge it in LTS 2020.06

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js [hunk 3]
```diff
@@ -421,9 +573,10 @@
       $typeMenu.attr("role", "menu");
 
       for ( var type in SearchEntity.TYPES) {
-        if (SearchEntity.TYPES[type][options.searchEntityType].enabled == false)
-          continue;
-
+      	if (SearchEntity.TYPES[type][options.searchEntityType].enabled == false || type.includes("_FALLBACK")) {
```
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:576]: Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640)

**Comment by yagee-de** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 2
           

Clones added
============
- mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js  16
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640&bid=20217210)

File: mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js (hunk 3 above)
Comment (yagee-de) [inline mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js:576]: Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6197640)

**Pull request closed without merging**